Client/Assets/CC_Script/CreditCardExtras.shared.mjs

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+class CreditCardExtras {
+  focusNextInputField() {
+    var inputFields = document.getElementsByTagName('input');
```

Comment:
`💡 Suggestion:` We can use the spread operator here to get an array instead of a nodelist. 
```suggestion
    const inputFields = [...document.getElementsByTagName('input')];
```